server/network/aoprotocol.py

```diff
@@ -597,7 +598,9 @@ def net_cmd_ms(self, args):
         self.client.area.set_next_msg_delay(len(msg))
         if(bool(self.server.config['log_chat'])):
             database.log_ic(self.client, self.client.area, showname, msg)
-
+        else:
+            database.log_buffer(f'[{self.client.area.abbreviation}] {showname}/{self.client.char_name}' +
+                          f'/{self.client.name} ({self.client.ipid}): {msg}')
```

Comment:
So if the server is not configured to log the chat, then log the chat. Got it.

Reply:
The whole idea was that the logs went to memory, rather a database making the server owner unable to read messages unless in case of abuse. I didn't intend to merge the feature over as I accidentally pushed the commits to this branch. Essentially I forgot about the PR since it nobody paid attention to it.

Reply:
I'll revert the commits later